timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -59,19 +60,19 @@ Create a candlestick visualization using the raw data in the table `stocks_real_
 ### Create a candlestick with raw data
 
   1.  In the query editor, use this SQL to query a Candlestick dataset using specified bucket interval:
-      ```sql
-      SELECT
-      time_bucket($bucket_interval, time) AS time,
-      symbol,
-      FIRST(price, time) AS "open",
-      MAX(price) AS high,
-      MIN(price) AS low,
-      LAST(price, time) AS "close",
-      FROM stocks_real_time
-      WHERE symbol =  $symbol
-      AND time > $__timeFrom()::timestamptz and time < $__timeTo()::timestamptz
-      GROUP BY symbol;
-      ```
+        ```sql
```

Comment:
I think what we meant here (at least what I meant!) is to indent lines that don't start new clauses, rather than indenting the entire statement. Makes it easier to scan the query and identify the important parts! The query further down is an example